Framework/Core/test/test_AnalysisDataModel.cxx

```diff
@@ -34,7 +34,7 @@ BOOST_AUTO_TEST_CASE(TestJoinedTables)
 
   TableBuilder trackParCovBuilder;
   auto trackParCovWriter = trackParCovBuilder.cursor<TracksCov>();
```

Comment:
I think this should be:

```C++
auto trackParCovWriter = trackParCovBuilder.cursor<StoredTracksCov>();
```

Reply:
Yes, and then no change in the call below is needed. Fixing it.